components/colorloupe/index.css

```diff
@@ -1,10 +1,10 @@
 :root {
   /* size is the same on desktop and mobile */
-  --spectrum-colorloupe-width: calc(
+  --spectrum-colorloupe-width-fixed: calc(
```

Comment:
Maybe `adjusted` would be better than `fixed` since fixed means something specific in CSS.